docs/feature-store/feature-sets.md

```diff
@@ -29,15 +29,20 @@ The feature set object contains the following information:
    
 ## Create a feature set
 
-Create a new {py:class}`~mlrun.feature_store.FeatureSet` with the base definitions:
+Create a {py:class}`~mlrun.feature_store.FeatureSet` with the base definitions:
 
-* **name**&mdash;The feature set name is a unique name within a project. 
-* **entities**&mdash;Each feature set must be associated with one or more index column. When joining feature sets, the entity is used as the key column.
-* **timestamp_key**&mdash;(optional) Used for specifying the time field when joining by time.
-* **engine**&mdash;The processing engine type:
+* **name** &mdash; The feature set name is a unique name within a project. 
+* **entities** &mdash; Each feature set must be associated with one or more index column. When joining feature sets, the key columns 
+   are determined by the the relations field if it exists, and otherwise by the entities.
+* **timestamp_key** &mdash; (optional) Used for specifying the time field when joining by time.
+* **engine** &mdash; The processing engine type:
    - Spark
    - pandas
-   - storey (some advanced functionalities are in the Beta state)
+   - storey. Default. (Some advanced functionalities are in the Beta state.)
+* **label_column** &mdash; Name of the label column (the one holding the target (y) values).
+* **relations** &mdash; (optional) Dictionary that indicates all of the relations between current feature set to other featuresets . It looks like: `{"<my_column_name>":Entity, ...}`. If the relation is None, and the `feature_set` 
```

Comment:
If the relation is None, and the `feature_set` relations is also None, the join is done on the entity
should become:
If the `feature_set` relations is None, the join is done based on `feature_set` entities

Reply:
@assaf758 fixed